src/main/resources/reference.conf

```diff
@@ -31,6 +31,35 @@ cassandra-journal {
   # Delay between connection retries
   connect-retry-delay = 5s
 
+  # Cassandra driver connection pool settings
+  # Documented at https://datastax.github.io/java-driver/features/pooling/
+  connection-pool {
+
+    # Create new connection threshold local
+    new-connection-threshold-local = 800
+
+    # Create new connection threshold remote
+    new-connection-threshold-remote = 200
+
+    # Connections per host core local
+    connections-per-host-core-local = 2
+
+    # CConnections per host max local
+    connections-per-host-max-local = 16
+
+    # Connections per host core remote
+    connections-per-host-core-remote = 2
+
+    # Connections per host max remote
+    connections-per-host-max-remote = 16
+
+    # Max requests per connection local
+    max-requests-per-connection-local = 32768
```

Comment:
I agree that it makes sense to increase this, since the default for v3 was chosen to be "compatible" (no surprise) with v2.